src/tracing_library.cpp

```diff
@@ -40,22 +37,13 @@ dd::Expected<dd::Tracer> TracingLibrary::make_tracer(
   config.agent.event_scheduler = std::make_shared<NgxEventScheduler>();
   config.integration_name = "nginx";
   config.integration_version = NGINX_VERSION;
+  config.service = "nginx";
 
   if (!nginx_conf.propagation_styles.empty()) {
     config.injection_styles = config.extraction_styles =
         nginx_conf.propagation_styles;
   }
 
-  if (nginx_conf.service_name) {
-    config.service = nginx_conf.service_name->value;
-  } else {
-    config.service = "nginx";
-  }
-
-  if (nginx_conf.environment) {
-    config.environment = nginx_conf.environment->value;
-  }
```

Comment:
we stop setting up the environment because we are not sure anymore that it will be set, am i right ?

Reply:
'cause no default environment and version.